### DISCUSSION TITLE
Problem on start/update

### DISCUSSION THREAD

**Discussion by MarksenDE** [projectdiscovery/nuclei#1453] [Q&A] [answered]
Hi guys,
i love so much your prog, great work! After i have installed it on latest Kali Version on Raspberry 4 it works great after typing this:
go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest

But when i run "nuclei -update" and process has finished i get this error:
![Bildschirmfoto vom 2022-01-03 21-28-46](https://user-images.githubusercontent.com/70513049/147977491-1fe290a2-3df8-4997-91ec-f9e4c3096863.png)

If i retry this regular install command everything works fine again. If i'm running a different go prog, this works.
What happened after "nuclei -update"?

Any ideas? Thanks if you can help

Mark

**Comment by ehsandeep** [comment, marked as answer]
@MarksenDE `nuclei -update` flag download the binary published at https://github.com/projectdiscovery/nuclei/releases/tag/v2.5.7, can you confirm if any of the listed binary is compatible with your system by downloading it manually?

**Reply by MarksenDE** [reply]
Hi ehsandeep, sorry for delay, i'll check it tomorrow asap

**Reply by MarksenDE** [reply]
Just checked it, was my fault ... wrong version, now it works. Really sorry
